api/sales_dashboard/templates/sales_dashboard/organisation.html

```diff
@@ -75,8 +75,8 @@ <h2>Subscription</h2>
                         <tr>
                             <td>{{ organisation.subscription.subscription_date }}</td>
                             <td>{{ organisation.subscription.cancellation_date }}</td>
-                            <td>{{ organisation.subscription.subscription_id }}</td>
-                            <td>{{ organisation.subscription.customer_id }}</td>
+                            <td><a target="_blank" href="https://flagsmith.chargebee.com/customers?view_code=all&Customers.search={{ organisation.subscription.customer_id }}">{{ organisation.subscription.customer_id }}</a></td>
```

Comment:
There's a chance these values could be null / invalid (in the case that an organisation is subscribed but pays via Xero for example). In which case you'll end up with an invalid link. Probably not a massive issue but pretty easy to clean up: 

```html
<td>{% if organisation.subscription.customer_id %}<a target="_blank" href="https://flagsmith.chargebee.com/customers?view_code=all&Customers.search={{ organisation.subscription.customer_id }}">{{ organisation.subscription.customer_id }}</a>{% endif %}</td>
<td>{% if organisation.subscription.subscription_id %}<a target="_blank" href="https://flagsmith.chargebee.com/subscriptions?view_code=all&Subscriptions.search={{ organisation.subscription.subscription_id }}">{{ organisation.subscription.subscription_id }}</a>{% endif %}</td>
```

Reply:
Ah yes - updated

Reply:
Sorry, just thinking about it more, probably the better option is: 

```html
{% if organisation.subscription.payment_method == "CHARGEBEE" %}...{% endif %}
```

I imagine some of the Xero subscriptions still have dummy data in these fields.

Reply:
OK tweaked